### PR TITLE
#75 Maze Program - Backtracking

### PR DESCRIPTION
**Pull request by anand-aman**
Issue #75 -   Maze Problem using Backtracking algorithmic technique

### CODE REVIEW

File: MazeBacktracking.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            System.out.println("Solution to Maze:\n");
+            printsolution();
+            System.out.println("\nHere, 1 represents the path followed.");
+        }
+        else
+            System.out.println("No possible solution");
+
+    }
+
+
```
Comment (rishabhgarg25699) [inline MazeBacktracking.java]: Remove extra input.

File: MazeBacktracking.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                return true;
+
+            //BACKTRACKING
+            solution[r][c] = 0;
+            return false;
+        }
+        return false;
+    }
+
+
```
Comment (rishabhgarg25699) [inline MazeBacktracking.java]: Remove extra space

File: MazeBacktracking.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
+//                                                 MAZE PROGRAM USING BACKTRACKING ALGORITHMIC APPROACH
+
+
+public class MazeBacktracking {
+
+    final static int SIZE = 5;
+
+    public static int[][] maze =
+    {
```
Comment (rishabhgarg25699) [inline MazeBacktracking.java]: Take input from user .

**Review by rishabhgarg25699 — CHANGES_REQUESTED** [review]
@anand-aman Do all changes

**Comment by anand-aman** [comment]
@rishabhgarg25699  I have made some changes. Now, maze will be taken as input from the user.

**Review by rishabhgarg25699 — APPROVED** [review]

**Pull request merged**